montepy/surfaces/cylinder_par_axis.py

```diff
@@ -42,19 +42,22 @@ def __init__(self, input: InitInput = None, number: int = None):
         self._radius = self._generate_default_node(float, None)
         super().__init__(input, number)
         ST = SurfaceType
-        if input:
-            if self.surface_type not in [ST.C_X, ST.C_Y, ST.C_Z]:
-                raise ValueError(
-                    "CylinderParAxis must be a surface of types: C/X, C/Y, C/Z"
-                )
-            if len(self.surface_constants) != 3:
-                raise ValueError(
-                    "CylinderParAxis must have exactly 3 surface_constants"
-                )
-            self._coordinates = self._surface_constants[0:2]
-            self._radius = self._surface_constants[2]
-        else:
-            self._surface_constants = [*self._coordinates, self._radius]
+        if self.surface_type not in [ST.C_X, ST.C_Y, ST.C_Z]:
+            raise ValueError(
+                "CylinderParAxis must be a surface of types: C/X, C/Y, C/Z"
+            )
+        if len(self.surface_constants) != 3:
+            raise ValueError("CylinderParAxis must have exactly 3 surface_constants")
+        self._coordinates = self._surface_constants[0:2]
+        self._radius = self._surface_constants[2]
+
+    @staticmethod
+    def _default_surf_type():
+        return "C/Z"
```

Comment:
I'm still not sure of default surface types

Get mnemonics from `SurfaceType` enum?